README.md

```diff
@@ -1,7 +1,75 @@
 # evotech
 Team Project for the CS2TP module
 
-## Credits
+# Running locally
+
+## Windows
+
+1. Install XAMPP
+    ```powershell
+    winget install -e --id ApacheFriends.Xampp.8.2
+    ```
+
+2. Clone the repository using `git clone`
+
+3. Change the DocumentRoot in `C:\xampp\apache\conf\httpd.conf` from `C:/xampp/htdocs` to the root of the repository
+    - There are two instances of DocumentRoot in the file, change both
+
+4. Start Apache and MySQL in XAMPP
+
+5. Open the following links in your browser:
+    - http://localhost/phpmyadmin
+    - http://localhost
+
+## macOS
+
+1. Install xcode command line tools
+    ```bash
+    xcode-select --install
+    ```
+
+2. Install brew
+    ```bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh) NONINTERACTIVE=1"
+    ```
+3. Install XAMPP
+    ```bash
+    brew install --cask xampp
+    ```
+    
+4. Clone the repository using `git clone`
+
+5. Change the DocumentRoot in `/Applications/XAMPP/xamppfiles/etc/httpd.conf` from `/Applications/XAMPP/xamppfiles/htdocs` to the root of the repository
+    - There are two instances of DocumentRoot in the file, change both
+
+6. Start Apache and MySQL in XAMPP
+
+7. Open the following links in your browser:
+    - http://localhost/phpmyadmin
+    - http://localhost
+    
+
+## Linux
+
+1. Install XAMPP
+    ```bash
+    wget https://www.apachefriends.org/xampp-files/8.2.12/xampp-linux-x64-8.2.12-0-installer.run
```

Comment:
this link gave a 404, xampp's website redirected me to sourceforge to dl it ( https://sourceforge.net/projects/xampp/files/XAMPP%20Linux/8.2.12/xampp-linux-x64-8.2.12-0-installer.run )